src/scss/pages/_contact.scss

```diff
@@ -1,130 +1,250 @@
-        }
-
-        &-button:hover {
-            background-color: $cozy-green-lite;
-        }
+        justify-content: space-around;
+        width: 286px;
+      }
+
+      &-icon {
+        height: 50px;
+        width: 50px;
+      }
+
+      &-custom {
+        color: $cozy-green-lite-darkest;
+        font-size: 20px;
+        font-weight: 600;
+        line-height: 24px;
+      }
+
+      &-text {
+        color: $black-color;
+        font-size: 16px;
+      }
+
+      &-inputs {
+        height: 214px;
+        width: 853px;
+      }
+
+      &-left {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        height: 145px;
+        justify-content: space-evenly;
+        width: 427px;
+      }
+
+      &-right {
+        align-items: center;
+        display: flex;
+        height: 145px;
+        justify-content: center;
+        width: 430px;
+      }
+
+      &-input {
+        background-clip: padding-box;
+        background-color: $home-white;
+        border: 1px solid $cozy-green-lite;
+        border-radius: 0.25rem;
+        color: $cozy-green-lite-darkest;
+        font-family: "Roboto", sans-serif;
+        font-size: 14px;
+        height: 32px;
+        line-height: 1.5;
+        padding: 0.375rem 0.75rem;
+        transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+        width: 349px;
+      }
+
+      &-message {
+        border: 1px solid $cozy-green;
+        border-radius: 0.25rem;
+        color: $cozy-green-lite-darkest;
+        font-family: "Roboto", sans-serif;
+        font-size: 14px;
+        height: 123px;
+        line-height: 1.5;
+        max-height: 123px;
+        max-width: 376px;
+        min-height: 100px;
+        min-width: 300px;
+        transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+        width: 376px;
+      }
+
+      &-footer {
+        align-items: center;
+        display: flex;
+        height: 69px;
+        justify-content: flex-end;
+        margin-right: 20px;
+        width: 100%;
+      }
+
+      &-button {
+        align-items: center;
+        background-color: $button-dark;
+        border: none;
+        border-radius: 7px;
+        color: $yellow-color;
+        display: flex;
+        font-family: "Roboto", sans-serif;
+        font-size: 14px;
+        font-weight: 800;
+        height: 47px;
+        justify-content: center;
+        line-height: 50px;
+        margin-right: 25px;
+        width: 169px;
+      }
+
+      &-button:hover {
+        background-color: $cozy-green-lite;
+      }
+    }
+
+    &-icon {
+      height: 50px;
+      width: 50px;
+
+      &-custom {
+        color: $cozy-green-lite-darkest;
+        font-size: 20px;
+        font-weight: 600;
+        line-height: 24px;
+      }
+    }
+
+    &-text {
+      color: $black-color;
+    }
+
+    &-inputs {
+      height: 214px;
+      width: 853px;
+    }
+
+    &-form {
+      &-box {
```

Comment:
Remove this class, there's no need of it, you can use the bootstrap class just for this.